samples/openthread/cli/CMakeLists.txt

```diff
@@ -12,3 +12,28 @@ find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
 project(openthread_cli)
 
 target_sources(app PRIVATE src/main.c)
+
+if(CONFIG_OPENTHREAD_SOURCES)
+  include(${ZEPHYR_NRF_MODULE_DIR}/cmake/ot_libs_configuration_write.cmake)
+
+  message(DEBUG "Building OT from sources, config file will be generated.")
+  ot_libs_configuration_write()
+
+  add_custom_target(install_openthread_libraries
```

Comment:
Much better looking.

The fact that this is an independent target, then why not have it available always, when building Openthread samples ?

Just like you have a `ninja flash`, `ninja run`, `ninja rom_report`, and similar targets available ?

Reply:
Tried to do it this time. Invoking with `west build -t install_openthread_libraries` works with all openthread examples.

Reply:
The dependencies are not correctly set up.
If you run:
```
$ cmake -GNinja -DBOARD=nrf52840dk_nrf52840 -Bbuild samples/openthread/cli/
$ ninja -Cbuild install_openthread_libraries
```
without building the sample, then you will see the following error:
```
FAILED: modules/nrf/CMakeFiles/install_openthread_libraries 
cd /projects/github/ncs/nrf/build/modules/nrf && /opt/cmake-3.17.1-Linux-x86_64/bin/cmake -E make_directory /projects/github/ncs/nrfxlib/openthread/lib/cortex-m4/soft-float/v1.1 && /opt/cmake-3.17.1-Linux-x86_64/bin/cmake -E copy /projects/github/ncs/nrf/build/modules/openthread/build/src/core/libopenthread-ftd.a /projects/github/ncs/nrfxlib/openthread/lib/cortex-m4/soft-float/v1.1 && /opt/cmake-3.17.1-Linux-x86_64/bin/cmake -E copy /projects/github/ncs/nrf/build/modules/openthread/build/src/core/libopenthread-mtd.a /projects/github/ncs/nrfxlib/openthread/lib/cortex-m4/soft-float/v1.1 && /opt/cmake-3.17.1-Linux-x86_64/bin/cmake -E copy /projects/github/ncs/nrf/build/modules/openthread/build/src/core/libopenthread-radio.a /projects/github/ncs/nrfxlib/openthread/lib/cortex-m4/soft-float/v1.1 && /opt/cmake-3.17.1-Linux-x86_64/bin/cmake -E copy /projects/github/ncs/nrf/build/modules/openthread/build/src/cli/libopenthread-cli-ftd.a /projects/github/ncs/nrfxlib/openthread/lib/cortex-m4/soft-float/v1.1 && /opt/cmake-3.17.1-Linux-x86_64/bin/cmake -E copy /projects/github/ncs/nrf/build/modules/openthread/build/src/cli/libopenthread-cli-mtd.a /projects/github/ncs/nrfxlib/openthread/lib/cortex-m4/soft-float/v1.1 && /opt/cmake-3.17.1-Linux-x86_64/bin/cmake -E copy /projects/github/ncs/nrf/build/modules/openthread/build/src/ncp/libopenthread-ncp-ftd.a /projects/github/ncs/nrfxlib/openthread/lib/cortex-m4/soft-float/v1.1 && /opt/cmake-3.17.1-Linux-x86_64/bin/cmake -E copy /projects/github/ncs/nrf/build/modules/openthread/build/src/ncp/libopenthread-ncp-mtd.a /projects/github/ncs/nrfxlib/openthread/lib/cortex-m4/soft-float/v1.1 && /opt/cmake-3.17.1-Linux-x86_64/bin/cmake -E copy /projects/github/ncs/nrf/build/modules/openthread/build/src/ncp/libopenthread-rcp.a /projects/github/ncs/nrfxlib/openthread/lib/cortex-m4/soft-float/v1.1 && /opt/cmake-3.17.1-Linux-x86_64/bin/cmake -E copy_directory /projects/github/ncs/modules/lib/openthread/zephyr/../include /projects/github/ncs/nrfxlib/openthread/include && /opt/cmake-3.17.1-Linux-x86_64/bin/cmake -E copy /projects/github/ncs/nrf/build/openthread_lib_configuration.txt /projects/github/ncs/nrfxlib/openthread/
Error copying file "/projects/github/ncs/nrf/build/modules/openthread/build/src/core/libopenthread-mtd.a" to "/projects/github/ncs/nrfxlib/openthread/lib/cortex-m4/soft-float/v1.1".
ninja: build stopped: subcommand failed.

```